Core/include/Acts/Utilities/TransformRange.hpp

```diff
@@ -8,6 +8,7 @@
 
 #pragma once
 
+#include <concepts>
```

Comment:
Maybe also include `<ranges>` for the view interface?

Reply:
Added now